includes/Story_Post_Type.php

```diff
@@ -269,11 +280,17 @@ public static function admin_enqueue_scripts( $hook ) {
 
 		$post             = get_post();
 		$story_id         = ( $post ) ? $post->ID : null;
-		$post_type_object = get_post_type_object( self::POST_TYPE_SLUG );
-		$rest_base        = ! empty( $post_type_object->rest_base ) ? $post_type_object->rest_base : $post_type_object->name;
 		$post_thumbnails  = get_theme_support( 'post-thumbnails' );
+		$rest_base        = self::POST_TYPE_SLUG;
+		$post_type_object = get_post_type_object( self::POST_TYPE_SLUG );
 
-		self::load_admin_fonts( $post );
+		if ( $post_type_object instanceof \WP_Post_Type ) {
+			$rest_base = ! empty( $post_type_object->rest_base ) ? $post_type_object->rest_base : $post_type_object->name;
```

Comment:
Q: Which case did you have in mind here in addition to `WP_Post_Type`? Or is this mainly for checking that `$post_type_object` is not `null`?

Reply:
Mainly for that, yes.

Reply:
Should it still continue with the rest of the logic, even if it's `null` (or `! instanceof \WP_Post_Type`)?

Reply:
See same discussion above. Cannot just stop the logic, but ideally should have some error catching.

Reply:
Yep, true that can't just stop.

Anyway, probably for now just adding a comment above that it's for ensuring that `$post_type_object` is not `null` could be enough for the purposes of this PR.